configsuite/extension/ext.py

```diff
@@ -0,0 +1,206 @@
+"""Copyright 2018 Equinor ASA and The Netherlands Organisation for
+Applied Scientific Research TNO.
+
+Licensed under the MIT license.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the conditions stated in the LICENSE file in the project root for
+details.
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+"""
+
+import os
+import importlib
+
+from docutils.parsers.rst import Directive
+from docutils.parsers.rst.directives import unchanged_required
+from docutils import nodes
+
+import configsuite
+from configsuite import MetaKeys as MK
+from configsuite import types
+from configsuite.extension.ext_nodes import (
+    visit_children,
+    visit_container,
+    visit_content,
+    visit_header,
+    depart_children,
+    depart_container,
+    depart_content,
+    depart_header,
+    cs_children,
+    cs_container,
+    cs_content,
+    cs_header,
+)
+
+BASIC_EXAMPLE = {
```

Comment:
What I would like to have is the possibility of having a configuration example in the documentation that can be copy pasted directly and used. How about allowing the user to specify a default configuration that is used as an example? Instead of adding another `Example` key.

And if it is not provided, we'll leave it with `string` and `integer` etc.

Reply:
Created: https://github.com/equinor/configsuite/issues/116